blogs.json

```diff
@@ -3868,6 +3868,13 @@
             "feed_url": "https://scotteg.github.io/feed.xml",
             "twitter_url": "https://twitter.com/scotteg"
           },
+          {
+            "title": "Scott Smith's Blog",
```

Comment:
@daveverwer I wasn't sure whether to put "Scott Smith Dev" here or not. It likely doesn't matter too much. If you think it does matter, maybe take a peak at [scottsmithdev.com](scottsmithdev.com) and see if you think `title` here should be changed. Thanks!

Reply:
It can be whatever you choose! Feel free to submit an update if you'd prefer something else.